test/integration/cloud-config-vcenter.ini.template

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
```

Comment:
Include instructions at the top of the file so users know what to do with this. You can use the other templates as starting point.